website/blog/2021-02-22-jest-27.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Jest 27: Leaving the past behind ⏩
```

Comment:
symbolic placement of the emoji on the very right unlike the other posts :P

Reply:
not sure if the title sounds a bit negative, it's not like Jest's past years are something you'd wanna leave behind :see_no_evil: 
but maybe I'm just overly cautious, with things like `jest-jasmine2` leaving them behind is def the goal :sweat_smile:

Reply:
I agree it's a bit negative. What about something forward looking like:
* New Foundations
* Looking ahead
* New Defaults for Jest #2 (Could add a nice callback to https://jestjs.io/blog/2016/09/01/jest-15 and specifically point out how we do this to keep Jest relevant and fast?)

Reply:
I was specifically trying to avoid 'foundations for the future' type of things because that was the title of the 25 blog post already :sweat_smile: but yeah I'll come up with something probably based on the third suggestion :+1:

Reply:
Changed :+1: